packages/preset-create-react-app/index.js

```diff
@@ -0,0 +1,77 @@
+const path = require('path');
+const { logger } = require('@storybook/node-logger');
+const mergePlugins = require('./helpers/mergePlugins');
+const reactScriptsPath = require('./helpers/reactScriptsPath');
+const processCraConfig = require('./helpers/processCraConfig');
+const checkPresets = require('./helpers/checkPresets');
+
+const SCRIPTS_PACKAGE_OPTION = 'scriptsPackageName';
+
+checkPresets();
+
+const resolveLoader = {
+  modules: ['node_modules', path.join(reactScriptsPath, 'node_modules')],
+};
+
+// Ensure that loaders are resolved from react-scripts.
+const managerWebpack = (webpackConfig = {}) => ({
+  ...webpackConfig,
+  resolveLoader,
+});
+
+// Update the core Webpack config.
+const webpack = (webpackConfig = {}, options = {}) => {
+  const storybookConfigDir = path.resolve(options.configDir);
+  let scriptsPath = reactScriptsPath;
+
+  // If the user has provided a package by name, try to resolve it.
+  if (options[SCRIPTS_PACKAGE_OPTION]) {
+    try {
+      scriptsPath = require.resolve(options[SCRIPTS_PACKAGE_OPTION]);
+    } catch (e) {
+      logger.warn(`A \`${SCRIPTS_PACKAGE_OPTION}\` was provided, but couldn't be resolved.`);
```

Comment:
Maybe throw here?

Reply:
Well, I wasn't sure. It may still work, as we might find the package anyway. So I feel it's safe to just warn?